masterlist.yaml

```
@@ -14609,6 +14618,67 @@ plugins:
         crc: 0x7E36F0E5
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 122
+        
+  # Additional Clockwork
+  - name: 'ClockworkCourierCounterspell.esp'
+    group: "Fixes & Resources"
+    clean:
+      - crc: 0x6abc5154
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+  - name: 'ClockworkShadowmarksCastleExt.esp'
+    msg:
+      - <<: *alreadyInX
+        subs: ["\"Additional Clockwork - Marked Shadows Merged\" (ClockworkShadowmarksMerged.esp)"]
+        condition: 'active("ClockworkShadowmarksMerged.esp")'
+    clean:
+      - crc: 0xc73f27d3
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
+  - name: 'ClockworkShadowmarksTermini.esp'
+    msg:
+      - <<: *alreadyInX
+        subs: ["\"Additional Clockwork - Marked Shadows Merged\" (ClockworkShadowmarksMerged.esp)"]
+        condition: 'active("ClockworkShadowmarksMerged.esp")'
+    clean:
+      - crc: 0x93ba7a0d
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
+  - name: 'ClockworkShadowmarksTravelRoom.esp'
+    msg:
+      - <<: *alreadyInX
+        subs: ["\"Additional Clockwork - Marked Shadows Merged\" (ClockworkShadowmarksMerged.esp)"]
+        condition: 'active("ClockworkShadowmarksMerged.esp")'
+    clean:
+      - crc: 0xedc64e4b
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
+  - name: 'ClockworkShadowmarksMerged.esp'
+    clean:
+      - crc: 0xb4c1e773
+        util: "SSEEdit v4.0.3"
+    msg:
+      - <<: *includesX
+        subs: ["\"Additional Clockwork - Marked Shadows - Castle Exterior\" (ClockworkShadowmarksCastleExt.esp)"]
+        condition: 'active("ClockworkShadowmarksCastleExt.esp")'
+      - <<: *includesX
+        subs: ["\"Additional Clockwork - Marked Shadows - Travel Room\" (ClockworkShadowmarksTravelRoom.esp)"]
+        condition: 'active("ClockworkShadowmarksTravelRoom.esp")'
+      - <<: *includesX
+        subs: ["\"Additional Clockwork - Marked Shadows - Termi\" (ClockworkShadowmarksTermini.esp)"]
+        condition: 'active("ClockworkShadowmarksTermini.esp")'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087' ]
```

Formatting changes + the `*includesX` messages are redundant with the above `*alreadyInX` messages:

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
    clean:
      - crc: 0xB4C1E773
        util: 'SSEEdit v4.0.3'
```